packages/config/src/projects/common/forceTransactions.ts

```diff
@@ -35,13 +35,20 @@ const STARKEX_PERPETUAL_WITHDRAW: ProjectTechnologyChoice = {
   ...WITHDRAW_OR_HALT,
   description:
     WITHDRAW_OR_HALT.description +
-    ' Perpetual positions can also be force closed before withdrawing.',
+    ' Perpetual positions can also be force closed before withdrawing, however this requires the user to find the counterparty for the trade themselves.',
   references: [
     {
       text: 'Censorship Prevention - StarkEx documentation',
       href: 'https://docs.starkware.co/starkex-v3/architecture/overview#8-censorship-prevention',
     },
   ],
```

Comment:
We need a link about the counterparty here

Reply:
Anyone can publish stateBatch as long as they have a bond. Currently BondManager that checks this is mocked (https://etherscan.io/address/0xCd76de5C57004d47d0216ec7dAbd3c72D8c49057#code) and vouch only for the OVM_Proposer EOA address

Reply:
https://docs.starkware.co/starkex-v3/starkex-deep-dive/regular-flows/flows-for-off-chain-accounts/forced-operations/perpetual-trading-forced-withdrawal-and-forced-trade#forced-trade